integration/run_test.go

```diff
@@ -336,6 +336,27 @@ func TestKritisPods(t *testing.T) {
 				}
 			},
 		},
+		{
+			description: "java-with-vuln-breakglass-deployment",
+			args: []string{"kubectl", "apply", "-f",
+				"integration/testdata/java/java-with-vuln-breakglass-deployment.yaml"},
+			deployments: []testObject{
+				{
+					name: "java-with-vuln-breakglass-deployment",
+				},
+			},
+			shouldSucceed: true,
+			dir:           "../",
+			cleanup: func(t *testing.T) {
+				cmd := exec.Command("kubectl", "delete", "-f",
+					"integration/testdata/java/java-with-vuln-breakglass-deployment.yaml")
+				cmd.Dir = "../"
+				output, err := integration_util.RunCmdOut(cmd)
+				if err != nil {
+					t.Fatalf("kritis: %s %v", output, err)
```

Comment:
Two comments: "kritis" doesn't seem like a very informative string here, when the error is relating to deletion. 

Also, is t.Fatalf the right choice? Is it possible for the test continue onward so that we don't have to play whack-a-mole with test failures?

Reply:
added more verbose logs to where "kritis:" used to be.

regarding t.Fatalf, the entire set of tests runs each time regardless of single failure, the t.Fatalf only stops the single test in the scope of the t.Run.